lib/SetAPI/genome/GenomeSetInterfaceV1.py

```diff
@@ -33,62 +50,73 @@ def save_genome_set(self, ctx, params):
             "set_info": info,
         }
 
-    def _validate_genome_set_data(self, data, save_search_set):
-        # TODO: add checks that only one copy of each genome data is in the set
-        if save_search_set:
-            if "elements" not in data:
-                raise ValueError(
-                    '"elements" list must be defined in data to save a KBaseSearch.GenomeSet'
-                )
-
-            if "description" not in data:
-                data["description"] = ""
-        else:
-            if "items" not in data:
-                raise ValueError(
-                    '"items" list must be defined in data to save a KBaseSets.GenomeSet'
-                )
-
-            # add 'description' and 'label' fields if not present in data:
-            for item in data["items"]:
-                if "label" not in item:
-                    item["label"] = ""
-            if "description" not in data:
-                data["description"] = ""
+    def _validate_save_set_params(
+        self: "GenomeSetInterfaceV1", params: dict[str, Any], save_search_set: bool
+    ) -> None:
```

Comment:
some extra excitement here -- another parameter!! Everything is pretty much the same apart from the name of the key under `data` that we're checking.